### ISSUE TITLE
Can't start server.py

### ISSUE THREAD

**Issue by rPman** [stablecabal/gyre#4]
On ubuntu 22.04 after installing and runing python server.py i see downloading files and next error:
```
apport_python_hook | OSError: Failed to load model hinters-uniformer-small. Failed attempts:
apport_python_hook |   - Path 'weights/hinters-uniformer-small-fp16' isn't a directory
apport_python_hook |   - Path 'weights/hinters-uniformer-small' isn't a directory
apport_python_hook |   - Error when trying to load weights from /home/user/.cache/huggingface/hub/models--halffried--gyre_hinters/snapshots/50822d9a0347f2ab0fa1a1c7b48addc8ab5c998a/uniformer_small/. libtorch_cuda_cu.so: cannot open shared object file: No such file or directory
```
there are .cache/huggingface/hub/models--halffried--gyre_hinters/snapshots/50822d9a0347f2ab0fa1a1c7b48addc8ab5c998a/uniformer_small/ exists files README.md upernet_global_small.safetensors upernet_global_small.yaml but symlink to ../../../blobs/...

after restarting the script, the files are deleted and re-downloaded.

full log with
# python3 server.py --log_level DEBUG --dep_log_level DEBUG
https://pastebin.com/JqwQUMAj

**Comment by rPman** [comment]
manual install, without miniconda
docker does not work too but other error, i create issue later

**Comment by hafriedlander** [comment]
Sorry for delay. I have confirmed it doesn't happen on Ubuntu 22.04 under WSL, so I need to create boot partition to test under "proper" conditions.

I assume you've set `export PIP_FIND_LINKS=https://download.openmmlab.com/mmcv/dist/cu116/torch1.12/index.html` as per the docs, can you try installing making sure that environment variable is _not_ set? It seems like there might be a problem with the distro binaries on some platforms (https://github.com/open-mmlab/mmdetection3d/issues/438)

(Next major version will be updated to PyTorch 2.0 and MMCV 2.0, so this won't be an issue there)

**Comment by MichaelKapustey** [comment]
Most likely got the same error.
Manual install, miniconda. Followed the installation steps

gyre.server        | OSError: Failed to load model hinters-uniformer-small. Failed attempts:
gyre.server        |   - Path 'weights\hinters-uniformer-small' isn't a directory
gyre.server        |   - Error when trying to load weights from C:\Users\thor\.cache\huggingface\hub\models--halffried--gyre_hinters\snapshots\50822d9a0347f2ab0fa1a1c7b48addc8ab5c998a\uniformer_small/. No module named 'mmcv._ext'
gyre.server        | Waiting for server to shutdown...
gyre.server        | All done. Goodbye.

**Comment by willzhou** [comment]
> 

Got the same error too, when the mmcv is v1.7.1.

Then, update mmcv to 2.0.0. The error becomes - 

rich.console       | OSError: Failed to load model hinters-uniformer-small. Failed attempts:
rich.console       |   - Path 'weights/hinters-uniformer-small-fp16' isn't a directory
rich.console       |   - Path 'weights/hinters-uniformer-small' isn't a directory
rich.console       |   - Error when trying to load weights from /home/willzhou/.cache/huggingface/hub/models--halffried--gyre_hinters/snapshots/50822d9a0347f2ab0fa1a1c7b48addc8ab5c998a/uniformer_small/. MMCV==2.0.0 is used but 
rich.console       | incompatible. Please install mmcv>=1.3.17, <=1.7.0.
gyre.server        | Waiting for server to shutdown...

**Comment by hafriedlander** [comment]
Yes, you won't be able to just force-install mmcv 2.0, there are several dependencies that break. Could either of you try the instructions from my previous comment (i.e., assuming you are installing after setting PIP_FIND_LINKS, try installing while making sure that environment variable is _not_ set)
